### ISSUE TITLE
Keycloak returns 'Invalid parameter: redirect_uri'

### ISSUE THREAD

**Issue by varolemre** [keycloak/keycloak-admin-ui#2371]
### Describe the bug

Hi. As usual, when i want to login to the app, i go to this URL:

Base url: http://mywebsite.com/auth/admin/master/console/

http://mywebsite.com/auth/realms/master/protocol/openid-connect/auth?client_id=security-admin-console&redirect_uri=http%3A%2F%2Fmywebsite.com%2Fauth%2Fadmin%2Fmaster%2Fconsole%2F%23%2F&state=b548d1a4-0411-46d9-88c5-8775b1105f5d&response_mode=fragment&response_type=code&scope=openid&nonce=4ceedc72-7f1d-41ad-9e59-365199966c93&code_challenge=1FQ_4rLLd5JMGCpvezsnx4D1uVABKvzRchXEhtrUa2Y&code_challenge_method=S256

And i login without any problem.
This is my security-admin-console settings belons master realm:
![Screenshot from 2022-03-30 09-17-42](https://user-images.githubusercontent.com/40075749/160765300-fbcd22fb-865d-4bc6-9986-3dd33edc2a68.png)

Then, I created a different realm and a user belong to this realm. 
Base url: http://mywebsite.com/auth/admin/test/console/

I have no problem when I try to login from this link until i change theme to keycloak.v2.

It says: 'Invalid parameter: redirect_uri'

And redirect to this URL:

http://mywebsite.com/auth/realms/master/protocol/openid-connect/auth?client_id=security-admin-console&redirect_uri=http%3A%2F%2Fmywebsite.com%2Fauth%2Fadmin%2Ftest%2Fconsole%2F%23%2F&state=8eda9d7c-e908-4d2d-9dbb-085d4b23d55e&response_mode=fragment&response_type=code&scope=openid&nonce=e227131d-6d77-43ba-95d4-ec91901a18a7&code_challenge=TKIH3x_qeRp_NwNsuMle-3IQ13Z0t9V7UW4GvAOMIQ8&code_challenge_method=S256

This is my security-admin-console settings belongs test realm:
![Screenshot from 2022-03-30 09-22-31](https://user-images.githubusercontent.com/40075749/160764933-4baa300b-86af-49d5-bcfb-92b74b190a34.png)

Any help would be greatly appreciated.

### Version

15.0.2

### Expected behavior

Should be able to login to different realm without any issues

### Anything else?

I don't have a problem working in localhost. I add ' ?realm="test" parameter to url, and it works. There is a code like this in environment.ts :

```
const realm =
  new URLSearchParams(window.location.search).get("realm") ?? "master";
```

**Comment by jonkoops** [comment]
This repository is for the development of the new Admin Console. If you are having issues with another component please report your issue on the [main repository](https://github.com/keycloak/keycloak).

**Comment by varolemre** [comment]
This issue is about new admin console. Because it happens only i change theme to keycloak.v2 ( new admin console ) .
 @edewit

**Comment by jonkoops** [comment]
Oh my bad, I went by the screenshots included in this issue which are from the old console. Re-opening.

**Comment by jonkoops** [comment]
@varolemre can you confirm this is an issue on the latest version of Keycloak when building it with the new Admin Console?

**Comment by ssilvert** [comment]
@varolemre When you created the new realm "test", it should have automatically set your Valid Redirect URIs to "/admin/test/console/*".

Is that not what happened?

Also, I could not reproduce the error you are seeing.  Can you try out the latest nightly build?
https://github.com/keycloak/keycloak/releases/tag/nightly

I'm not sure how you got a base URL that points to master realm with a redirect to test realm.  Can you give the exact steps you used to reproduce that?

**Comment by varolemre** [comment]
@ssilvert  Sure, i can give the exact steps one by one. 

Step 1: Im login to my master realm:
![Screenshot from 2022-03-30 17-39-27](https://user-images.githubusercontent.com/40075749/160861536-101a92ef-3927-41f9-bb14-56e394a14442.png)

Step 1.a : I logged in successfully:
![Screenshot from 2022-03-30 17-43-09](https://user-images.githubusercontent.com/40075749/160862390-ba21bc08-36ed-4897-9e4b-2a5fdc0c1207.png)

Step 2: I created a new realm:
![Screenshot from 2022-03-30 17-44-00](https://user-images.githubusercontent.com/40075749/160862542-bf04680a-f4db-4f91-b808-4e69ebcf8424.png)

Step 3: These are my new realms clients:
![Screenshot from 2022-03-30 17-46-04](https://user-images.githubusercontent.com/40075749/160863060-6d558b60-3120-478d-a5a7-e7c3bb59cfdf.png)

Step 3.a : security-admin-console settings:
![Screenshot from 2022-03-30 17-47-12](https://user-images.githubusercontent.com/40075749/160863275-b09c6c65-a095-4771-b7c3-2b116cd4d8e6.png)

Step 3.c : I created an user belongs demo realm.

Step 4 : I go to http://mywebsite.com/auth/admin/demo/console/
So far, no problem
![Screenshot from 2022-03-30 17-49-13](https://user-images.githubusercontent.com/40075749/160863731-5b8045e5-e9c2-4093-bf66-72029c3233c6.png)

Step 5 : I login to my newuser account in demo realm :
![Screenshot from 2022-03-30 17-52-36](https://user-images.githubusercontent.com/40075749/160864395-19c85906-1cef-4357-b76a-30436a793fba.png)

Step 6 :  I logged in successfully:
![Screenshot from 2022-03-30 17-52-36](https://user-images.githubusercontent.com/40075749/160864551-31c6366f-57d9-4dad-b9ef-c597e3365f2e.png)

Step 7: Now, i want to change my keycloak theme v1 to v2.

**Comment by ssilvert** [comment]
Thanks.  I followed your steps and it works fine for me.  But I'm on the latest build and I'm seeing that you are on Keycloak 15. 

Can you try a nightly build?
https://github.com/keycloak/keycloak/releases/tag/nightly

**Comment by jonkoops** [comment]
Can this problem be verified in the nightly build? Otherwise I see no reason to keep this issue open.

**Comment by edewit** [comment]
We still have this problem it will only be resolved if we update keycloak-js where this is a fix

**Comment by edewit** [comment]
fixed with: #2490

**Comment by ssilvert** [comment]
@varolemre Thanks for reporting this.  I'm going to close now because it should be fixed.
